inst/example_workflow.R

```diff
@@ -0,0 +1,55 @@
+#file with example workflow for basic package functionality - will be helpful later tu build vignettes
+
+git_stats <- GitStats$new()
+
+git_stats$set_connection(
+  api_url = "https://api.github.com",
+  token = Sys.getenv("GITHUB_PAT"),
+  owners_groups = c("openpharma", "r-world-devs")
+)
+
+git_stats
+
+# examples for getting repos
+
+git_stats$get_repos_by_owner_or_group()
+
+git_stats$get_repos_by_codephrase("Dashboard")
+
+git_stats$get_repos_by_codephrase("Dashboard",
+                                  language = "Python")
+
+# set your team members
+
+git_stats$set_team(team_name = "RWD",
+                   "galachad",
+                   "krystian8207",
+                   "kalimu",
+                   "marcinkowskak",
+                   "Cotau",
+                   "maciekbanas")
+
+git_stats$get_repos_by_team("RWD")
+
+# you can plot repos sorted by last activity
+
+git_stats$plot_repos()
```

Comment:
it rises warning, though plots are generated and look nice:
`Warning messages:
1: In RColorBrewer::brewer.pal(N, "Set2") :
  minimal value for n is 3, returning requested palette with 3 different levels

2: In RColorBrewer::brewer.pal(N, "Set2") :
  minimal value for n is 3, returning requested palette with 3 different levels`

Reply:
I created an issue for that #35